flink-connectors/flink-sql-connector-hive-2.3.9/src/main/resources/META-INF/NOTICE

```diff
@@ -24,8 +27,15 @@ the Apache Software License 2.0 (http://www.apache.org/licenses/LICENSE-2.0.txt)
 - javax.jdo:jdo-api:3.0.1
 - joda-time:joda-time:2.8.1
 - net.sf.opencsv:opencsv:2.3
-- org.apache.avro:avro-mapred:1.7.7
-- org.apache.avro:avro:1.7.7
+- org.apache.avro:avro-guava-dependencies:1.8.2
+- org.apache.avro:avro-mapred:1.8.2
+- org.apache.avro:avro:1.8.2
+- org.apache.calcite:calcite-core:1.10.0
+- org.apache.calcite:calcite-druid:1.10.0
+- org.apache.calcite:calcite-linq4j:1.10.0
+- org.apache.calcite.avatica:avatica:1.8.0
+- org.apache.calcite.avatica:avatica-metrics:1.8.0
```

Comment:
`org.apache.calcite.avatica:avatica-metrics:jar:1.8.0` don't include in the `hive-exec` shaded jar?

Reply:
indeed, its part of avatice (seems to be a fat-jar)